From cc35cc5e0db2b921b9104922b85e8e8c7c797d5e Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Fri, 13 Apr 2018 23:29:07 +0200
Subject: [PATCH 14/15] Fix linking of `cctest` in mingw-w64

We do not have .obj files. Really. Also, it seems that we somehow got
the wrong path for the object files of the node target, let's fix them
at the same time.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 node.gyp | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/node.gyp b/node.gyp
index 8a7af9bf..1ff15292 100644
--- a/node.gyp
+++ b/node.gyp
@@ -570,9 +570,8 @@
         'OBJ_SUFFIX': 'o',
         'conditions': [
           ['OS=="win"', {
-            'OBJ_PATH': '<(OBJ_DIR)/node',
-            'OBJ_GEN_PATH': '<(OBJ_DIR)/node',
-            'OBJ_SUFFIX': 'obj',
+            'OBJ_PATH': '<(OBJ_DIR).target/node/src',
+            'OBJ_GEN_PATH': '<(OBJ_DIR).target/node/gen',
           }],
           ['OS=="aix"', {
             'OBJ_PATH': '<(OBJ_DIR)/node_base/src',
-- 
2.17.0.windows.1

